src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -1051,6 +1051,10 @@
     <value>Enable experimental text rendering engine</value>
     <comment>An option to enable an experimental text rendering engine</comment>
   </data>
+  <data name="Profile_VtPassthrough.Header" xml:space="preserve">
+    <value>Enable experimental virtual terminal passthrough</value>
+    <comment>An option to enable experimental virtual terminal passthrough connectivity option with the underlying ConPTY</comment>

```

Comment:
This comment could use some work -- it is supposed to be read by a human translator to help them understand what they are translating